onnxruntime/core/providers/dnnl/dnnl_op_manager.cc

```diff
@@ -6,64 +6,64 @@
 
 namespace onnxruntime {
 DnnlOpManager::DnnlOpManager() {
```

Comment:
Why these capabilities are commented? I am not sure if that is the part of the graph design, hence the question.

Reply:
Just for testing, you can ignore it